odxtools/endofpdufield.py

```diff
@@ -156,10 +156,12 @@ def _resolve_odxlinks(self, odxlinks: OdxLinkDatabase) -> None:
     def _resolve_snrefs(self, diag_layer: "DiagLayer") -> None:
         """Recursively resolve any short-name references"""
         if self.structure_snref is not None:
-            self._structure = diag_layer.data_object_properties[self.structure_snref]
+            self._structure = diag_layer.diag_data_dictionary_spec.data_object_props[
+                self.structure_snref]
 
         if self.env_data_desc_snref is not None:
-            self._env_data_desc = diag_layer.data_object_properties[self.env_data_desc_snref]
+            self._env_data_desc = diag_layer.diag_data_dictionary_spec.data_object_props[
+                self.env_data_desc_snref]
```

Comment:
diag_data_dictionary_spec makes a simple property access breaks multiple lines

Maybe put a local variable `spec = diag_layer.diag_data_dictionary_spec` to simplify code?

Reply:
good point. fixed: 76b4606189d7f645741d3348843fca72741a7aed